.codeqlmanifest.json

```diff
@@ -17,6 +19,9 @@
         "ruby/extractor-pack/codeql-extractor.yml",
         "ql/extractor-pack/codeql-extractor.yml"
     ],
+    "ignore": [
```

Comment:
Let's just delete this. I suspect it is no longer needed.

Reply:
Thanks.  Done.